data/transactions/logic/assembler.go

```diff
@@ -157,7 +157,6 @@ func (ops *OpStream) Intc(constIndex uint) {
 	} else {
 		ops.trace("intc %d %d", constIndex, ops.intc[constIndex])
 	}
-	ops.tpush(StackUint64)
```

Comment:
Does this happen somewhere else now?

Reply:
Yes, a whole bunch of opcodes were simplified by moving the type stack handling into checkArgs.  It used to only check that the stack was the right shape at opcode entry, but not it also performs the update for the shape of the stack when the opcode finishes. Since we now proceed after errors in order report more, it's quite nice that it does type stack management at the opcode start. Even if the opcode ends up erroring out - the type stack will still be updated so we won't get cascading errors as we continue assembling.  Only opcodes that return different types (field access mainly) need to do explicit management now.